keep/api/routes/workflows.py

```diff
@@ -450,7 +451,8 @@ def get_workflow_execution_status(
             status_code=404,
             detail=f"Workflow execution {workflow_execution_id} not found",
         )
-
+    
+    workflow_execution.results = json.loads(workflow_execution.results)
```

Comment:
its not how to do it, you can do it on the model itself with sqlmodel/sqlalchemy